src/Form/Control/Input.php

```diff
@@ -231,7 +231,7 @@ protected function renderView(): void
         }
 
         // set template
-        $this->template->setHtml('Input', $this->getInput());
+        $this->template->dangerouslySetHtml('Input', $this->getInput());
```

Comment:
@georgehristov is this correct? Can you please review this PR if there are more simillar issues?